src/main/java/net/fabricmc/fabric/api/fluid/FluidMeter.java

```diff
@@ -10,28 +26,26 @@
  * 
  * By design, it serves only to handle the accounting of contents.
  * Assumes all units tracked are for the <em>same</em> fluid. (Tracking
- * multiple fluids in the same device will require more than one {@link FluidMeter}
- * but otherwise does not know anything about the tracked fluid or its behavior.<p>
+ * multiple fluids in the same device will require more than one {@link FluidMeter}.)
+ * Does not know anything about the tracked fluid or its behavior.<p>
  * 
  * The enclosing device may constrain the units allowed for transfer in or out
  * depending on the fluid itself and expected use cases.<p>
  * 
  * The enclosing device is also responsible for transfer semantics such as capacity limits, 
- * simulated operations, etc. (TODO: those are likely to be a separate part of this API.)
- * 
- * TODO: static factory for creating/loading instances.
+ * simulated operations, etc.
  */
 public interface FluidMeter {
```

Comment:
I'm just realizing that `FluidMeter` might not be the best name, given that the world is measured in meters in statistics. Maybe `FluidVolume`?

Reply:
Good point. I like FluidVolume, or perhaps just Volume?  I am also wondering if FluidVolume should exist as a separate entity from an actual FluidStorage or whatever we call it that provides the actual fluid handling functionality or just be baked in.  My intuition is that it should remain distinct in the API but can't yet articulate why.

Reply:
I mean it's like the difference between `ItemStack` and `Inventory` really.